pkg/controllers/netpol/utils.go

```diff
@@ -142,3 +142,43 @@ func getPodIPv4Address(pod podInfo) (string, error) {
 	}
 	return "", fmt.Errorf("pod %s has no IPv4Address", pod.name)
 }
+
+// safeJoin joins the namespace and name, ensuring that the result is less than or equal to 48 characters
+func safeJoin(namespace string, name string) string {
+	if (len(namespace) + len(name)) < 48 {
+		return namespace + "/" + name
+	}
+
+	// We must create at least one substring
+	if len(namespace) < 24 {
+		lengthSubString := (48 - len(namespace) - 1)
+		return namespace + "/" + name[0:lengthSubString]
+	}
+
+	if len(name) < 24 {
+		lengthSubString := (48 - len(name) - 1)
+		return namespace[0:lengthSubString] + "/" + name
+	}
+
+	//If we arrive here, both are over 24 characters
+	return namespace[0:23] + "/" + name[0:23]
+}
+
+// getIptablesNFlogLimit reads the annotations setting the nflog limit and limit-burst config
+// "k3s.io/netpol-nflog-limit" and "k3s.io/netpol-nflog-limit-burst"
+func getIptablesNFlogLimit(annotations map[string]string) (string, string) {
+	defaultLimit := "10/minute"
+	defaultLimitBurst := "10"
+
+	limit, ok := annotations["k3s.io/netpol-nflog-limit"]
+	if !ok {
+		limit = defaultLimit
+	}
+
+	limitBurst := annotations["k3s.io/netpol-nflog-limit-burst"]
```

Comment:
Do you think this will be upstreamed to flannel? If so, should we use the flannel annotation prefix here instead of k3s.io?